lib/galaxy/webapps/galaxy/api/history_contents.py

```diff
@@ -202,6 +202,19 @@ def download_dataset_collection(self, trans, id, history_id=None, **kwd):
         """
         return self.service.download_dataset_collection(trans, id)
 
+    @expose_api_anonymous
+    def download_dataset_collection_async(self, trans, id, history_id=None, **kwd):
+        """
+        GET /api/histories/{history_id}/contents/dataset_collections/{id}/download_async
+        GET /api/dataset_collection/{id}/download_async
+
+        Return to a short term storage token to monitor download of the collection.
```

Comment:
```suggestion
        Return a short term storage token to monitor download of the collection.
```
Can we rename this to `prepare_download` ?

Reply:
Sure - I can do that. Or maybe ``prepare_collection_download`` I suppose.